internal/quickstart/create_flag.go

```diff
@@ -67,6 +85,14 @@ func (m createFlagModel) View() string {
 	style := lipgloss.NewStyle().
 		MarginLeft(2)
 
+	if m.showSuccessView {
+		successMessage := fmt.Sprintf("Flag %q created successfully!", m.flag.name)
+		if m.existingFlagUsed {
+			successMessage = fmt.Sprintf("Using existing flag %q for setup.", m.flag.name)
+		}
+		return successMessage + " Press enter to continue."
+	}
```

Comment:
@dbolson updated the name here - is this doing too much/do I need another message type here to pass around?

Reply:
Yeah, maybe we should have a message for when we create a new flag and one where the flag already exists.

Reply:
ah I meant for this comment to go on the Update step where I was overloading one message type, but think I've managed to fix